app/assets/stylesheets/article-show.scss

```diff
@@ -31,6 +31,97 @@
   height: 10px;
 }
 
+.mod-actions-menu {
+  position: fixed;
+  top: var(--header-height);
+  right: 0;
+  height: calc(100vh - 56px);
+  z-index: 200;
+  transform: translateX(360px);
+  transition: var(--transition-props);
+  &.showing {
+    transform: initial;
+  }
+
+  iframe {
+    height: 100%;
+    border: none;
+    box-shadow: -4px 0px 15px rgba(0, 0, 0, 0.1);
+    width: 360px;
+  }
+}
+
+.mod-actions-menu-btn {
```

Comment:
Why not using one of our existing buttons?

```
<button class="crayons-btn crayons-btn--icon-rounded crayons-btn--l"><svg ...></button>
<!-- ^that's also available as preact component -->
```

Reply:
I think this was before we had the buttons in, will update 👍

Reply:
Hmm so it would be a bit bigger if I used the smallest crayons button... here's a preview, with the original being on the left:
![2020-05-12 12 04 48](https://user-images.githubusercontent.com/17884966/81717707-de847400-9448-11ea-967a-83107631b4c9.gif)

@lisasy what do you think, regarding the sizing and background color?

Reply:
I think using the "Crayons" button with the larger size (the one on the right) is a good approach. Thanks for looking me in @Zhao-Andy and @ludwiczakpawel